gapic-generator-java/src/test/java/com/google/api/generator/engine/writer/goldens/GrpcServiceClientWithNestedClassImport.golden

```diff
@@ -31,19 +31,26 @@ import javax.annotation.Generated;
  * resources such as threads. In the example above, try-with-resources is used, which automatically
  * calls close().
  *
- * <p>The surface of this class includes several types of Java methods for each of the API's
```

Comment:
Looks like we are generating less info now? Are these information still useful?

Reply:
We replaced that generic paragraph with examples specific to the service under each type (e.g. flattened, request object, and callable). So still the same information, but with specific examples.